code/modules/religion/darkness_sect.dm

```diff
@@ -0,0 +1,133 @@
+/**** Darkness Sect ***/
+
+/datum/religion_sect/darkness_sect
+	name = "Dark City"
+	desc = "A sect dedicated to the darkness."
+	convert_opener = "Turn out the lights, acolyte, and let the darkness cover the world!<br>The altar and manifested obelisks will generate favor from being in darkness."
+	alignment = ALIGNMENT_NEUT
+	favor = 100 //Starts off with enough favor to make an obelisk
+	//No desired items, favor is gained through obelisks and the altar in darkness
+	rites_list = list(/datum/religion_rites/extend_darkness,/datum/religion_rites/darkobelisk, /datum/religion_rites/darkconversion)
+	altar_icon_state = "convertaltar-dark"
+	var/light_reach = 1
+	var/light_power = 0
+	var/list/obelisks
+
+/datum/religion_sect/darkness_sect/sect_bless(mob/living/blessed, mob/living/user)
+	return TRUE
+
+/datum/religion_sect/darkness_sect/on_select(atom/religious_tool, mob/living/user)
+	. = ..()
+	if(!religious_tool || !user)
+		return
+	religious_tool.AddComponent(/datum/component/dark_favor, user)
+
+/**** Rites ****/
+
+#define DARKNESS_INVERSE_COLOR "#88ff88" //The color of light has to be inverse, since we're using negative light power
+
+/datum/religion_rites/darkconversion
+	name = "Shadowperson Conversion"
+	desc = "Converts a humanoid into a shadowperson, a race blessed by darkness."
+	ritual_length = 30 SECONDS
+	ritual_invocations = list("Let the darkness seep into you...",
+						"... And cover you, envelope you ...",
+						"... And make you one with it ...")
+	invoke_msg = "... And let you be born again!"
+	favor_cost = 500
+
+/datum/religion_rites/darkconversion/perform_rite(mob/living/user, atom/religious_tool)
+	if(!ismovable(religious_tool))
+		to_chat(user, "<span class='warning'>This rite requires a religious device that individuals can be buckled to.</span>")
+		return FALSE
+	var/atom/movable/movable_reltool = religious_tool
+	if(!movable_reltool)
+		return FALSE
+	if(!LAZYLEN(movable_reltool.buckled_mobs))
+		if(!movable_reltool.can_buckle) //yes, if you have somehow managed to have someone buckled to something that now cannot buckle, we will still let you perform the rite!
+			to_chat(user, "<span class='warning'>This rite requires a religious device that individuals can be buckled to.</span>")
+			return FALSE
+		to_chat(user, "<span class='warning'>This rite requires an individual to be buckled to [movable_reltool].</span>")
+		return FALSE
+	return ..()
+
+/datum/religion_rites/darkconversion/invoke_effect(mob/living/user, atom/religious_tool)
+	if(!ismovable(religious_tool))
+		CRASH("[name]'s perform_rite had a movable atom that has somehow turned into a non-movable!")
+	var/atom/movable/movable_reltool = religious_tool
+	if(!movable_reltool?.buckled_mobs?.len)
+		return FALSE
+	var/mob/living/carbon/human/human2darken = locate() in movable_reltool.buckled_mobs
```

Comment:
how does this list work? I am pretty sure you can just loop it instead of locating